### PR TITLE
Fix arrange time

### PR DESCRIPTION
**Pull request by BenZacs**
I alredy fix the issue that event can be created with arrange time in the past

### CODE REVIEW

File: Kvent/views.py [hunk 1]
```diff
@@ -61,18 +61,22 @@ def create_event(request):
...
+                        long_description = form.data.get('long_description')
+                        event = Event(event_name = event_name, location=location, short_description = short_description, long_description = long_description, arrange_time = arrange_time, number_people = number_people,full=False, photo=photo, user=request.user)
+                        event.save()
+                        messages.success(request, f"You've created the {event_name} event!")
+                        return HttpResponseRedirect(reverse('index'))
+                    else:
+                        messages.warning(request, "Arrangement date must be in the future!")
+                except:
+                    messages.warning(request, f"You should input the date and time as format!")
+                    return render(request, 'Kvent/create-event-page.html', {'form': form})
```
Comment (bleachjade) [inline Kvent/views.py:79]: Please try to pull it and test on your machine if this works or not.

**Review by bleachjade — COMMENTED** [review]
I've pulled and tested your code but it has some error when the user types date and time that doesn't match the format so I fixed some code by using try-except, please review this change.

**Comment by forfeen** [comment]
I've already checked it. The function is working correctly.

**Comment by JirawadeeSampusri** [comment]
I have already run server and the date event box can not add for today and past on create event page. also, the alert message to warning the wrong date.

**Review by JirawadeeSampusri — APPROVED** [review]

**Pull request merged**